support-frontend/assets/pages/supporter-plus-landing/components/summaryTsAndCs.test.tsx

```diff
@@ -43,14 +43,23 @@ describe('Summary Ts&Cs Snapshot comparison', () => {
 		`summaryTs&Cs for $productKey With ratePlanKey $activeRatePlanKey renders correctly`,
 		({ productKey, activeRatePlanKey }) => {
 			// Act
+			const countryGroupId =
+				productKey === 'DigitalSubscription' && activeRatePlanKey === 'Monthly'
+					? 'UnitedStates'
+					: 'GBPCountries';
+			const currency =
+				productKey === 'DigitalSubscription' && activeRatePlanKey === 'Monthly'
+					? 'USD'
+					: 'GBP';
```

Comment:
I wonder if it'd make sense to add currency and country group ID to the test table above - it seems more explicit that way (rather than deriving from the product and rate plan). What do you think?